src/Storages/MergeTree/MergeTreeIndexText.h

```diff
@@ -128,44 +139,39 @@ struct PostingsSerialization
         RawPostings = 1ULL << 0,
         /// If set, the posting list is embedded into the dictionary block to avoid additional random reads from disk.
         EmbeddedPostings = 1ULL << 1,
+        /// If set, the posting list is serialized as a single block.
```

Comment:
L. 137 says the cardinality for raw postings is <= 16, however `MAX_CARDINALITY_FOR_RAW_POSTINGS` is defined as 12.